inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/HudiLoadNode.java

```diff
@@ -72,6 +79,7 @@ public Map<String, String> tableOptions() {
         Map<String, String> options = super.tableOptions();
         options.put("connector", "inlong-hudi");
```

Comment:
Hi, connector is 'inlong-hudi', but I can't find code in sort-connector-hudi

Reply:
Thks, I have already dealt with it in another PR, will close this PR.